sql/core/src/test/scala/org/apache/spark/sql/SparkSessionBuilderSuite.scala

```diff
@@ -281,4 +281,23 @@ class SparkSessionBuilderSuite extends SparkFunSuite with BeforeAndAfterEach {
       ()
     }
   }
+
+  test("SPARK-33139: Test spark.sql.legacy.allowModifyActiveSession") {
+    Seq(true, false).foreach { allowModifyActiveSession =>
+      val session = SparkSession.builder()
+        .master("local")
+        .config(SQLConf.LEGACY_ALLOW_MODIFY_ACTIVE_SESSION.key, allowModifyActiveSession.toString)
```

Comment:
nit: allowModifyActiveSession is enough here

Reply:
thanks

Reply:
done